components/__tests__/AnimationsShowcase.test.tsx

```
@@ -0,0 +1,276 @@
+  })
+
+  describe('hover interactions', () => {
+    it('handles hover events on scale interaction element', () => {
+      render(<AnimationsShowcase />)
+
+      // Find the scale hover element by its text content
+      const scaleElement = screen.getByText('Hover to scale').closest('div')
+      expect(scaleElement).toBeInTheDocument()
+
+      // Test hover events
+      fireEvent.mouseEnter(scaleElement!)
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)